web/src/site-admin/SiteAdminReportBugPage.tsx

```diff
@@ -117,23 +119,29 @@ export const SiteAdminReportBugPage: React.FunctionComponent<Props> = ({ isLight
                     support@sourcegraph.com.
                 </div>
             </div>
-            <DynamicallyImportedMonacoSettingsEditor
-                value={
-                    allConfig
-                        ? JSON.stringify(
-                              monitoringStats ? { ...allConfig, ...monitoringStats } : { ...allConfig, alerts: null },
-                              undefined,
-                              2
-                          )
-                        : ''
-                }
-                jsonSchema={allConfigSchema}
-                canEdit={false}
-                height={800}
-                isLightTheme={isLightTheme}
-                history={history}
-                readOnly={true}
-            />
+            {isLoading ? (
+                <LoadingSpinner className="icon-inline mt-2" />
+            ) : (
+                <DynamicallyImportedMonacoSettingsEditor
+                    value={
+                        allConfig
+                            ? JSON.stringify(
+                                  monitoringStats
+                                      ? { ...allConfig, ...monitoringStats }
+                                      : { ...allConfig, alerts: null },
+                                  undefined,
+                                  2
+                              )
+                            : ''
```

Comment:
`allConfig` is always defined at this point afaict. If you inline the `isLoading` condition, TypeScript should be able to tell
```suggestion
                            JSON.stringify(
                                  monitoringStats
                                      ? { ...allConfig, ...monitoringStats }
                                      : { ...allConfig, alerts: null },
                                  undefined,
                                  2
                            )
```

Reply:
It does indeed 😁 fixed in 6f92aa6db1cef25148ec4c70a38c8797715409a6